packages/client/src/compose/ReplyForm.tsx

```diff
@@ -79,13 +110,39 @@ export default function ReplyForm({
             value={value}
             placeholder="Write your reply here."
           />
+          {Attachments(attachments, classes.attachment)}
         </CardContent>
-        <CardActions>
+        <CardActions className={classes.actionRow}>
           <Button type="submit" disabled={replyResult.loading}>
             Send Reply
           </Button>
+          <input
+            className={classes.input}
+            id="add-attachment-button"
+            multiple
+            type="file"
+            onChange={handleAttachments}
+          />
+          <label htmlFor="add-attachment-button">
+            <Tooltip title={"Add Attachment"}>
+              <IconButton component="span">
+                <AttachFileIcon />
+              </IconButton>
+            </Tooltip>
+          </label>
```

Comment:
As you mentioned, we want to create reusable components for this stuff. We can work together on implementing the compound components pattern.